lib/inkjet/version.rb

```diff
@@ -1,3 +1,3 @@
 module Inkjet
-  VERSION = '0.0.6'
+  VERSION = '0.0.7'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.